commit 94febb4483269542fdb11a805ba9449ab04a5875
Author: Anthony Mallet <anthony.mallet@laas.fr>
Date:   Sun Nov 20 19:01:05 2011 +0100

    Use LIBS instead of LDFLAGS when looking for libtcl in configure.ac

--- configure~	2011-06-16 17:17:42.000000000 +0200
+++ configure	2011-11-22 16:01:29.870379024 +0100
@@ -11301,7 +11301,7 @@
 ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
    ac_tmp_cppflags=$CPPFLAGS; CPPFLAGS="${CPPFLAGS} ${TCL_INCLUDE_SPEC}"
-   ac_tmp_ldflags=$LDFLAGS; LDFLAGS="${LDFLAGS} ${TCL_LIB_SPEC}"
+   ac_tmp_libs=$LIBS; LIBS="${LIBS} ${TCL_LIB_SPEC}"
    cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 #include "$tcl_test_include"
@@ -11328,7 +11328,7 @@
 rm -f core conftest.err conftest.$ac_objext \
     conftest$ac_exeext conftest.$ac_ext
    CPPFLAGS=$ac_tmp_cppflags
-   LDFLAGS=$ac_tmp_ldflags
+   LIBS=$ac_tmp_libs
 
    # Add libtool -R flag to the runtime path of tcl
    { $as_echo "$as_me:${as_lineno-$LINENO}: checking for a sed that does not truncate output" >&5
@@ -11510,9 +11510,9 @@
 ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
    ac_tmp_cppflags=$CPPFLAGS;
-   ac_tmp_ldflags=$LDFLAGS;
+   ac_tmp_libs=$LIBS;
    CPPFLAGS="${CPPFLAGS} ${TK_INCLUDE_SPEC} ${TCL_INCLUDE_SPEC}"
-   LDFLAGS="${LDFLAGS} ${TK_LIB_SPEC} ${TCL_LIB_SPEC}"
+   LIBS="${LIBS} ${TK_LIB_SPEC} ${TCL_LIB_SPEC}"
    cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 #include "$tk_test_include"
@@ -11539,7 +11539,7 @@
 rm -f core conftest.err conftest.$ac_objext \
     conftest$ac_exeext conftest.$ac_ext
    CPPFLAGS=$ac_tmp_cppflags
-   LDFLAGS=$ac_tmp_ldflags
+   LIBS=$ac_tmp_libs
 
    # Add libtool -R flag to the runtime path of tcl
    { $as_echo "$as_me:${as_lineno-$LINENO}: checking for a sed that does not truncate output" >&5
